src/BenchmarkDotNet/Portability/RuntimeInformation.cs

```diff
@@ -168,11 +179,15 @@ internal static Runtime GetCurrentRuntime()
                 return MonoRuntime.Default;
             if (IsFullFramework)
                 return ClrRuntime.GetCurrentVersion();
+            if (IsBenchmarkTargetWasm || IsWasm())
```

Comment:
`IsBenchmarkTargetWasm` should be removed for the reasons explained above

```suggestion
            if (IsWasm())
```